x-pack/packages/kbn-elastic-assistant/impl/connectorland/connector_setup/index.tsx

```diff
@@ -151,7 +151,9 @@ export const useConnectorSetup = ({
     () =>
       conversation.messages.slice(0, currentMessageIndex + 1).map((message, index) => {
         const isUser = message.role === 'user';
-
+        const timestamp = `${i18n.CONNECTOR_SETUP_TIMESTAMP_AT}: ${new Date(
+          message.timestamp
+        ).toLocaleString()}`;
```

Comment:
nit: We didn't to this initially, but probably should be using the `dateFormat:tz`/`dateFormat` as configured in Kibana Advanced Settings. We don't inject those settings into the assistant package at the moment, so no need to address in this PR, but just wanted to mention.